### ISSUE TITLE
Function output-isolation has no parameter named $span

### ISSUE THREAD

**Issue by jedfoster** [at-import/Singularity#209]
Testing against LibSass v3.2.0-beta.6 (via node-sass v3.0.0-beta.7) I'm getting this error:

```
cat test/fixtures/Singularity.gs.scss | node-sass --include-path lib/sass_modules/vendor/singularity/stylesheets/ --include-path lib/sass_modules/
{
  "message": "function output-isolation has no parameter named $span",
  "column": 87,
  "line": 59,
  "file": "lib/sass_modules/vendor/singularity/stylesheets/singularitygs/_api.scss",
  "status": 1
}
```

My test Sass: 

``` scss
@import "breakpoint";
@import "singularitygs";

$grids: 1 3 5 7 9;
$gutters: 1/3;

#foo {
  @include grid-span(2, 3, $output-style: 'isolation', $options: 'both');
}
#foo {
  @include isolation-span(2, 3, 'both');
}

#bar {
  @include grid-span(2, 3, (2 8 2 1), .25, 'isolation');
}
#bar {
  @include isolation-span(2, 3, $grid: (2 8 2 1), $gutter: .25);
}

#baz {
  @include grid-span(2, 3, (2 8 2 1), .25, 'right');
}
#baz {
  @include isolation-span(2, 3, 'right', (2 8 2 1), .25);
}
```

This _may_ be an issue with LibSass and not Singularity, but I'm not familiar enough with your code to know for sure.

**Comment by Snugug** [comment]
Was an issue with that release of Libsass. Defect reported, acknowledge, and fix was merged for final 3.2 release. Node Saas 3.0 should be coming out any time now and when it does I can confirm it's been fixed in the final version, but it's a libsass issue

**Comment by Snugug** [comment]
Interestingly, works fine in 3.1
